stable/prometheus-cloudwatch-exporter/templates/deployment.yaml

```diff
@@ -23,6 +23,9 @@ spec:
         checksum/config: {{ include (print $.Template.BasePath "/configmap.yaml") . | sha256sum }}
         checksum/secrets: {{ include (print $.Template.BasePath "/secrets.yaml") . | sha256sum }}
     spec:
+      {{- with .Values.priorityClassName }}
```

Comment:
shouldn't this be wrapped in an if statement ?

Reply:
`with` acts like an `if`, if the value of the expresion is `empty`

https://golang.org/pkg/text/template/
> {{with pipeline}} T1 {{else}} T0 {{end}}
> 	If the value of the pipeline is empty, dot is unaffected and T0
> 	is executed; otherwise, dot is set to the value of the pipeline
> 	and T1 is executed.